<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SEClass</class>
 <widget class="QMainWindow" name="SEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>酒吧博弈</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>445</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="gobarButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Go Bar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stayhomeButton">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stay Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showstatusButton">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Show Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SCORE：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="score_label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>9</y>
      <width>800</width>
      <height>295</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>15</number>
    </attribute>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>34</y>
      <width>81</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="player_label">
       <property name="text">
        <string>Player</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label1">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label2">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label3">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label4">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label5">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label6">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label7">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="robot_label8">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="name_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_input_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入玩家名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_text">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>446</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ROUND：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="round_label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SE.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>gobarButton</sender>
   <signal>clicked()</signal>
   <receiver>SEClass</receiver>
   <slot>goBarButtonFuc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>821</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>name_button</sender>
   <signal>clicked()</signal>
   <receiver>SEClass</receiver>
   <slot>setnameButtonFuc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>863</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>continueButton</sender>
   <signal>clicked()</signal>
   <receiver>SEClass</receiver>
   <slot>continueButtonFuc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showstatusButton</sender>
   <signal>clicked()</signal>
   <receiver>SEClass</receiver>
   <slot>changeStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>838</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stayhomeButton</sender>
   <signal>clicked()</signal>
   <receiver>SEClass</receiver>
   <slot>stayHomeButtonFuc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>845</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goBarButtonFuc()</slot>
  <slot>changeStatus()</slot>
  <slot>setnameButtonFuc()</slot>
  <slot>continueButtonFuc()</slot>
  <slot>stayHomeButtonFuc()</slot>
 </slots>
</ui>
